CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ endif(CUDA_STATIC_RUNTIME)
 
 target_link_libraries(rmm INTERFACE rmm::Thrust)
 target_link_libraries(rmm INTERFACE spdlog::spdlog_header_only)
+target_link_libraries(rmm INTERFACE dl)
```

Comment:
The official cmake way is `${CMAKE_DL_LIBS}` instead of `dl`.

But I'm also wondering why it's needed -- it doesn't seem like you're using `dlopen`/`dlclose`?

Reply:
`dl` is needed by `boost::stacktrace`.

Reply:
Thank you, I've added this.